src/models/SubconModels.js

```diff
@@ -2,7 +2,11 @@ const SubconModels = {
   processResponseData(response) {
     const dataArray = response.data;
 
-    const processedData = dataArray.map((config) => ({
+    // Filter out entries with id === 1
```

Comment:
please add proper comment. Something like "Filter out entries with id === 1 as it is representing Budget"